src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -37,17 +37,22 @@
 
 ;;;; Handlers
 
-(handlers/register-handler-db
+(defn choose-address-and-name [db address name]
+  (assoc db :wallet/send-transaction
+         {:to-address address
```

Comment:
i believe `update` should be used here, because we don't want to replace `:wallet/send-transaction` map, but just update two fields